chapters/executors.rst

```diff
@@ -0,0 +1,113 @@
+==========================================
+A Guide to Jina Executors
+==========================================
```

Comment:
maybe just to clarify, how do you think we rename it to something like `A guide to how to use Jina Executors`, to avoid get confuse with `A guide to Customise Jina Executor`